data/hai/hai reveal 5 philosophers.txt

```diff
@@ -264,10 +334,23 @@ mission "Hai Reveal [B04] The Road Not Taken"
-			`	"What I've just send you is the modern standard presentation for a set of coordinates that were listed in the notation. Now it's possible there's some kind of code or cipher, in which case they'd be wrong, but if they are indeed just direct from the notation, then they indicate a system in the vicinity of Syndicate space that is not on the regular star maps. It must require a jump drive to access, but fortunately I know you happen to have one."`
-			`	She shrugs. "Of course I have no idea how they'd have a jump drive, but that's not my problem. Whether you find something or not come back and see me. If that turns up bust I found an online community who seem to be a support group for people who've had family members kidnapped. The missing person reports for some of them are suspiciously close matches or share features with those in the files you left for me. Not sure if it'll reveal anything but we can look into it if you don't find anything. I haven't looked deeply though, it just came up yesterday while I was chasing down loose threads."`
-			`She picks up a bag and starts putting some items from her desk into it. "In any event, I really must go. Check out those coordinates, and then come see me. Fly safe Captain!"`
+			`	"So, the notation is actually quite dense, and is mostly some probably meaningless prose about something called 'Elenchus'. It sounds aspirational. However, it also refers to a secret gateway. I'm sending you some coordinates." You receive a simple text message a moment later with some system coordinates in it.`
+			choice
+				`	"What is Elenchus?"`
+				`	"I've never seen this system on a star map. What do you expect to find there?"`
+			`	Giti ignores your question and continues. "What I've just send you is the modern representation of coordinates in the notation. Now it's possible there's some kind of code or cipher, in which case they'd be wrong, but if they are direct from the notation, they indicate a system in Syndicate space that is not on the regular star maps. It must require a jump drive to access, but fortunately I know you happen to have one."`
+			`	She shrugs. "Of course, I have no idea how they'd have a Jump Drive, but that's not my problem. Whether you find something or not, come back and see me. If that turns up bust, I found an online community who seem to be a support group for people who've had family members kidnapped. The missing person reports for some of them are suspiciously close matches or share features with those in the files you left for me. I haven't looked deeply though, it just came up yesterday while I was chasing down loose threads."`
+			choice
+				`	"How do you know I have a Jump Drive?"`
```

Comment:
```suggestion
				`	"How do you know I have a jump drive?"`
```
I think this is public knowledge. The player does jump to Sol with one during the Free Worlds campaign.

Reply:
I have swapped that with the next question, so the default option is not to ask a rhetorical question.